snapshots/tests.js

```diff
@@ -1206,17 +1208,71 @@ test(
   `
 // Text zu Sprache
 
-sage [Hallo] :: tts
-ändere die Stimme zu (Alt v) :: tts
-setze Sprache auf (English v) :: tts
+sage [Hallo]
+ändere die Stimme zu (Alt v)
+setze Sprache auf (English v)
 
 // Übersetzung
 
-(übersetze [Hallo] nach (Malaysisch v) :: translate)
-(Sprache :: translate)
+(übersetze [Hallo] nach (Malaysisch v))
+(Sprache)
 
+// Makey Makey
+Wenn [nach links nach oben nach rechts v] der Reihe nach gedrückt
+
+// micro:bit
+Wenn Knopf [A v]gedrückt wird
+<Knopf [A v] gedrückt? >
+Wenn [bewegt v]
+zeige [heart v] an
+zeige Text [Hallo!] an
+zeige nichts an
+Wenn [beliebiger v] geneigt
+<[beliebig v] geneigt?>
+(Neigungswinkel [nach vorne v])
+Wenn Pin [0 v] angeschlossen ist
+
+// EV3
+drehe Motor [A v] für (1)Sekunden rechtsherum
+drehe Motor [A v]für (1)Sekunden linksherum
+setze Leistung von Motor [A v] auf (100)%
+
+// Commented out until disambiguation issue can be fixed
+// (Position von Motor [A v])
```

Comment:
Can we leave this uncommented? It's nice to be able to look at the snapshots and see what's left to do, if that makes any sense!
```suggestion
(Position von Motor [A v])
```

Reply:
Ah okay. It will be red for a while but...